include/deal.II/lac/schur_complement.h

```diff
@@ -129,36 +129,45 @@ DEAL_II_NAMESPACE_OPEN
  * An illustration of typical usage of this operator for a fully coupled
  * system is given below.
  * @code
- *    #include<deal.II/lac/schur_complement.h>
- *
- *    // Given BlockMatrix K and BlockVectors d,F
- *
- *    // Decomposition of tangent matrix
- *    const auto A = linear_operator(K.block(0,0));
- *    const auto B = linear_operator(K.block(0,1));
- *    const auto C = linear_operator(K.block(1,0));
- *    const auto D = linear_operator(K.block(1,1));
- *
- *    // Decomposition of solution vector
- *    auto x = d.block(0);
- *    auto y = d.block(1);
- *
- *    // Decomposition of RHS vector
- *    auto f = F.block(0);
- *    auto g = F.block(1);
- *
- *    // Construction of inverse of Schur complement
- *    const auto prec_A = PreconditionSelector<...>(A);
- *    const auto A_inv = inverse_operator<...>(A,prec_A);
- *    const auto S = schur_complement(A_inv,B,C,D);
- *    const auto S_prec = PreconditionSelector<...>(D); // D and S operate on
- * same space const auto S_inv = inverse_operator<...>(S,...,prec_S);
- *
- *    // Solve reduced block system
- *    auto rhs = condense_schur_rhs (A_inv,C,f,g); // PackagedOperation that
- * represents the condensed form of g y = S_inv * rhs; // Solve for y x =
- * postprocess_schur_solution (A_inv,B,y,f); // Compute x using resolved
- * solution y
+ * #include<deal.II/lac/schur_complement.h>
+ *
+ * // Given BlockMatrix K and BlockVectors d,F
+ *
+ * // Decomposition of tangent matrix
+ * const auto A = linear_operator(K.block(0,0));
+ * const auto B = linear_operator(K.block(0,1));
+ * const auto C = linear_operator(K.block(1,0));
+ * const auto D = linear_operator(K.block(1,1));
+ *
+ * // Decomposition of solution vector
+ * auto x = d.block(0);
+ * auto y = d.block(1);
+ *
+ * // Decomposition of RHS vector
+ * auto f = F.block(0);
+ * auto g = F.block(1);
+ *
+ * // Construction of inverse of Schur complement
+ * const auto prec_A = PreconditionSelector<...>(A);
+ * const auto A_inv = inverse_operator<...>(A,prec_A);
+ * const auto S = schur_complement(A_inv,B,C,D);
+ *
+ * // D and S operate on same space
+ * const auto S_prec = PreconditionSelector<...>(D);
+ * const auto S_inv = inverse_operator<...>(S,...,prec_S);
+ *
+ * // Solve reduced block system
+ * // PackagedOperation that represents the condensed form of g
```

Comment:
Another one I did not catch before: I think that the comment starting with `PackagedOperation` refers to the line containing `condense_schur_rhs`

Reply:
Let me be more specific. Here is the original version:

```
 *    // Solve reduced block system
 *    auto rhs = condense_schur_rhs (A_inv,C,f,g); // PackagedOperation that represents the condensed form of g
 *    y = S_inv * rhs; // Solve for y
 *    x = postprocess_schur_solution (A_inv,B,y,f); // Compute x using resolved solution y
```

so the `y = ` line is duplicated.